Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml

```diff
@@ -2537,7 +2537,9 @@ script:
       description: "QueryId. Example: pq3be5e2747f716cxxxxxxxxxxxxx20a0."
     description: Automate a power query and return the query results. (The maximum timeout of 300 seconds is allowed.)
     name: sentinelone-get-power-query-results
-    outputs: []
+    outputs:
+    - contextPath: SentinelOne.PowerQuery.[result_index]
```

Comment:
Why not add all the object fields from the result as context output fields?
It should be much clearer like this

Reply:
Results fields are dynamic as per query created by the customer
The API will return the results as a list of dynamic fields - if it was static we could have added them like other commands

Reply:
![image](https://github.com/user-attachments/assets/207c955e-a517-4ae4-9c4d-ee16918433e0)
like this
The command we used to have earlier for this which was deprecated in last version contained the same - we missed adding it in last version - hence we are doing it now

Reply:
@yaakovpraisler could you please let me know if there are any further action items on our side?
we want to get this published as soon as possible, a customer is waiting on this, thank you!

Reply:
@saurabh-metron It can be a list of objects and still you can add the fields as context output, it should work.
If all the results are dynamic and there are no constant fields then ok, but if there are some static fields I would suggest adding them in the context.

Reply:
@yaakovpraisler in this case all of the are dynamic, the fields inside the object (inside the list) depends on what query (column) you have used.

Reply:
Ok - here is an example proving the fields are totally dynamic and customers will have to access this context according to their queries. We cannot provide anything static inside the object returned from API - its totally dynamic and depends on what power_query you write - which is totally in control of customers, its not random, a customer would know what they are doing.

Example 1:
Query: `event.time = * | columns event.type, account.id, agent.version | limit 2`
![image](https://github.com/user-attachments/assets/47ebdf04-e2fd-4af6-a970-46fbc7d5a443)


Example 2:
Query: `event.time = * | columns event.time, agent.uuid, site.id | limit 2`
![image](https://github.com/user-attachments/assets/3fcb3ad3-3075-40dd-badf-f6b457705009)

You can see how the fields vary and dynamic in nature
Hope this is making sense? Please let me know if there any further questions? or please help me get this published, thank you!

Reply:
Also - this should be `list` rather than `string` as we will send list of object
| SentinelOne.PowerQuery.ResultIndex | string | Result from the query in array string format | 

So should i mention this instead in README? like this 
`| SentinelOne.PowerQuery.ResultIndex | list | Result from the power query in list of objects format | `

Reply:
If it's a list then yes, it should be "List".

Reply:
Done